test/test_scanner.py

```diff
@@ -792,12 +792,29 @@ def _file_test(self, url, filename, package, version):
                     # "CVE-2018-1060"
                 ],
             ),
+            (
+                "test-python-2.7.out",
+                "python",
+                "2.7.0",
+                [
+                    # check for known CVEs in this version
+                    "CVE-2019-9948",  # Vulnerability type: Bypass
+                    # "CVE-2019-10160",
+                    #  Failing for above CVE, though its present on NVD: https://nvd.nist.gov/vuln/detail/CVE-2019-10160
```

Comment:
Now that we've confirmed that this likely isn't a good cve to use, let's remove it from the comments.